### PR TITLE
emberfest 2023 stage1

### PR DESCRIPTION
**Pull request by oscard0m**
# 📓 Description
- Move 2022 to /2022
- Create **frontpage**  for 2023 (stage 1) **with sections**
- Create **Imprint**, **Cancellation** and **Code of Conduct** pages
- Add link to **Sponsorships** prospectus

# TODO
- [x] Set up the page as per the designs with the **landing page** (make sure to use the "Step 1" design for the header for now – "Step 2" is only released once we reveal the location)
- [x] Move the existing 2022 page into its folder like the past years (the 2022 page should be completely self-contained and not depend on anything that's not in the /2022 folder so we avoid accidentally changing it later) 
- [x] Create the rest of pages: **code of conduct**, **cancellation policy**, **imprint**
- [x] Add Sections in frontpage: **about**, 2022 video, past speakers, past sponsors, stay up to date

## PR feedback
| Task | Scope | Assignee | status |
|--------|--------|--------|--------|
| The circle button text doesn't look vertically centered. | **frontpage** |  | **<kbd>Done ✅</kbd>** 48a241539aa346844d8a521cd0e4a0f84e071f16 |
| Schedule links in 2022 still not working | **/2022** |  | **<kbd>Done ✅</kbd>** 9c0d44f8a9a25f89fc45cfd1662bdd64d21c18c2 |
| Old sites have a 2023 link in the footer as well | **/2022** |  | **<kbd>Done ✅</kbd>** 9c0d44f8a9a25f89fc45cfd1662bdd64d21c18c2 |
| The footer should not remove space from the viewport height of the hero section | **frontpage** |  | **<kbd>Done ✅</kbd>** dbb25fe9a4dc408678fb30fdb59737a4350ae748 |
| Replace `hover` links styles | **frontpage** |  | **<kbd>Done ✅ </kbd>** 8ef29ad9a4830e28f2bf5acfad796635d4518ade |
| Improve arrows hack to make the style look the same. Use 2 SVGs instead | **frontpage** |  | **<kbd>Done ✅ </kbd>** 24663127b5f23c879f8c083b4906262b9c5aecc3 |
| `<footer>` first element on each row does not have a `-` | **frontpage** |  | **<kbd>TODO 🔮  </kbd>** |
| Replace `glasses-small` image with the one provided [here](https://shared-assets.adobe.com/link/1a48c974-c6cc-4c69-5572-4f68c4ea2aaa) for better resolution. | **frontpage** |  | **<kbd>Done ✅ </kbd>** e1bcfcfefa8797ab16d37904e9dc5d38c7361786 |
| Add extra padding to `sidepages` push down the footer | **sidepages** |  | **<kbd>Done ✅ </kbd>** c6ccc0384ef7eaa1e271b0164b16c21bdf2e0156 |
| Try to serve different images depending on the viewport size instead of the hack of `display:none` (mode details [here](https://web.dev/serve-responsive-images/#serve-multiple-image-versions)) for **sidepages**| **sidepages** |  | **<kbd>Done ✅  </kbd>** 0462ab16699f2f6c22bc8b4090a8ed460c2fa60c |
| Use [THIS](https://www.sketch.com/s/150ed20a-d7c3-46a1-8151-89fba683acae/a/ZVJJlR3) scale of the background element based on the viewport height | **sidepages** |  | **<kbd>Done ✅ </kbd>** 3aaaecee84110d6165d41afe99a7a8612fd21f5c |
| Add extra padding/margin in sidepages for tablet viewport sizes | **sidepages** |  | <kbd>Done ✅</kbd> 0bf88136679273992f6bbedb545645c08ae12f2e |
| Closing horizontal line in footer should look the same as the arrow | **footer** |  | <kbd>Done ✅</kbd> 5b9f1da17a259e69ce7c938564d5a534cc71ae46 |
| About | **frontpage** |  | <kbd>Done ✅</kbd> 003c5f669edabf67680778e295521f0e79ea7e59 |
| 2022 video | **frontpage** |  | <kbd>Done ✅</kbd> 6be4f3fc2f46f3d06bd738529731cc5ca9f50469|
| Past Sponsors | **frontpage** |  | <kbd>Done ✅</kbd> 2befa149fa3ab3c9572f2a6038522da65801a8ce |
| Past Speakers | **frontpage** |  | <kbd>Done ✅ </kbd> e936eea2214a7853290a9f4eea266675a9618ee2 |
| Stay up to date | **frontpage** |  | <kbd>Done ✅ </kbd> 97ba406477ffc3a49439189cae5b1299f5cda14d |

## 🔬 Preview Links
- [🔗 2022](https://deploy-preview-385--silly-leavitt-2e0668.netlify.app/2022)
- [🔗 Frontpage](https://deploy-preview-385--silly-leavitt-2e0668.netlify.app/)
- [🔗 Code of conduct](https://deploy-preview-385--silly-leavitt-2e0668.netlify.app/code-of-conduct/)
- [🔗 Cancellation Policy](https://deploy-preview-385--silly-leavitt-2e0668.netlify.app/cancellation/)
- [🔗 Imprint](https://deploy-preview-385--silly-leavitt-2e0668.netlify.app/imprint/)

### Design 🎨 
[Link to sketch files](https://www.sketch.com/s/150ed20a-d7c3-46a1-8151-89fba683acae/v/l4xROO/p/CBD1D9FE-707B-435F-A92E-EEF811AFB0CC/canvas?posX=-21946.89703431879&posY=-5167.054341845888&zoom=0.10000000149011612)

# Context
Closes first part of #384

### CODE REVIEW

File: css/app.scss [hunk 1]
```diff
@@ -5,920 +5,104 @@
...
-  background-image: url(/images/background.svg);
-  background-repeat: no-repeat;
-  background-position: bottom center;
-  background-size: cover;
-  text-align: center;
-  padding-top: 134px;
-  padding-bottom: 380px;
+  min-height: 667px;
+  min-width: 350px;
+  height: 100vh;
```
Comment (oscard0m) [inline css/app.scss:16]: This depends on the answer to this comment: https://github.com/emberfest/emberfest.github.io/issues/384#issuecomment-1539093352

Also, when we go for **Step 2**, we will remove this and let the sections grow the content of the frontpage, but for **Step 1**, we want to enforce a `100vh`

**Comment by oscard0m** [comment]
@marcoow can we include @davidhajba in `github.com/emberfest` so we can request PR reviews from him?

**Comment by marcoow** [comment]
invited David ✅

**Review by marcoow — CHANGES_REQUESTED** [review]
The 2022 page doesn't work – see at https://deploy-preview-385--silly-leavitt-2e0668.netlify.app/2022/

Didn't check in detail but obvious problems on first impression are links don't work and font is missing

**Comment by oscard0m** [comment]
> The 2022 page doesn't work – see at https://deploy-preview-385--silly-leavitt-2e0668.netlify.app/2022/
> 
> Didn't check in detail but obvious problems on first impression are links don't work and font is missing

@marcoow I applied the fix for the broken links, only the ones for the `schedule` were broken as far as I could find.

Issue with the fonts broken fixed ✅

**Review by marcoow — COMMENTED** [review]
* schedule links in 2022 still not working, other links might not work neither, didn't check entire site
* all of the existing sites now need a 2023 link in the footer as well

**Comment by davidhajba** [comment]
hey @oscard0m thanks for the updates - GitHub invite accepted

* vertical alignment for the Button is good (I think the font is a bit tricky, same for me)

* the footer (in the Sketch version) should not remove space from the viewport height of the hero section, 
it should come underneath as an extra scrollable space. What do you think? 

* thank you for the hover idea, my concern with it - as the orange is a very dominant color on other objects on the page, and turning the button text color orange too kind of hides the button between the other elements instead highlighting it. If I can make a suggestion for hower action [HERE](https://www.sketch.com/s/150ed20a-d7c3-46a1-8151-89fba683acae/v/38YOOD/p/BE993FE2-E578-440C-A931-8457C6430A8A/canvas?posX=-533&posY=81&zoom=1) is my idea

* the arrows look broken on my side, could you please check?
Screenshot 2023-05-10 at 09 50 14

* extra '-' at the end of a row  -  if it's cannot be removed, sure it shouldn't be a problem

**Comment by davidhajba** [comment]
@oscard0m 
could you pls change the 'glasses-small' asset to the 'glasses-small-2' I just uploaded 

although the resolution is correct it is not sharp enough, I doubled the resolution (the size is still tiny)

assets are [HERE](https://shared-assets.adobe.com/link/1a48c974-c6cc-4c69-5572-4f68c4ea2aaa)

**Comment by oscard0m** [comment]
@davidhajba 

> * the footer (in the Sketch version) should not remove space from the viewport height of the hero section,
>   it should come underneath as an extra scrollable space. What do you think?

I applied a `100vh` for the hero section + the content filling a `100%`. I would like @marcoow's opinion on this change, he has pointed out the issue of having a fixed height of elements as a danger practice for different viewport sizes.

> * thank you for the hover idea, my concern with it - as the orange is a very dominant color on other objects on the page, and turning the button text color orange too kind of hides the button between the other elements instead highlighting it. If I can make a suggestion for hower action [HERE](https://www.sketch.com/s/150ed20a-d7c3-46a1-8151-89fba683acae/v/38YOOD/p/BE993FE2-E578-440C-A931-8457C6430A8A/canvas?posX=-533&posY=81&zoom=1) is my idea

Applied the `:hover` style you proposed for the **Book Your Ticket** button. I kept the hover style for the footer links. Is that ok?

> * the arrows look broken on my side, could you please check?

I tried a new approach. It looks better but not perfect. Let me know if it's enough.

> * extra '-' at the end of a row  -  if it's cannot be removed, sure it shouldn't be a problem

I posted this in #frontend because I'm out of ideas. The fact of having different viewport sizes makes this element to fluid in different ways so it's hard to target when a link is in row1 or row2 and if it's the first element of the row or not.

**Comment by davidhajba** [comment]
FRONTPAGE - looks almost perfect, only 2 little details

* the line attached to the arrowhead in the footer should be 1,5 px instead of 2, so it will be the same boldness with the head and will be perfectly aligned to the center vertically. The other line in the footer without arr arrowheads is currently 3px, pls change it to 1,5 px too
Screenshot 2023-05-12 at 13 57 34

* can the '21-22 SEP. 2023 / LOCATION TBA' be closer to the top as it is [HERE](https://www.sketch.com/s/150ed20a-d7c3-46a1-8151-89fba683acae/a/oMbvAyD) 

CODE OF CONDUCT, CANCELLATION POLICY, IMPRINT
* the changes I suggested in the other ticked are not updated yet, could you please check?

**Comment by davidhajba** [comment]
hey @inesilva 
on my side, I do not see these changes
maybe because of Safari or I'm not sure

Screenshot 2023-05-15 at 10 40 48

**Comment by inesilva** [comment]
> Details

I know what the problem is.
You are probably accessing through the links in the description, and they are outdated. I'll update them, but if you access through here, it is always the representation of the latest changes:
image

**Comment by davidhajba** [comment]
@inesilva - sure I can use the other link, thank you.

 Only a few comments from my side, 
 
  * if I scroll further on the top & the bottom of the home page an extra darker color appears -> would be good to keep this in the same color as the background // and if we want to make it perfect see the background SVG too without cropping it //
 
Screenshot 2023-05-15 at 13 25 02
 
Screenshot 2023-05-15 at 13 24 30

* The H1 font is an italic on the sub-pages <- this should be Extra-Bold (not italic)

Screenshot 2023-05-15 at 13 30 58

* The SVG  background element on the home page is perfect scale. 
On the sub-pages should be exactly the same too. (currently, it's different)

* Currently in the footer the arrow lines are 2px and the arrow-head SVGs are 1,5px.
We can go with the 2px arrow lines, but please change the arrow-heads for 2px too, I saved the assets [HERE](https://shared-assets.adobe.com/link/77f70c01-0762-46ff-460a-2bdf26bcd46c) 

Could you please help with these?
Other than these tiny details everything looks very beautiful 👌

File: _includes/hero.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    <h3>About Ember &#123;&#123;fest&#125;&#125;</h3>
+    <p>
+      EmberFest is the European Community Ember Conference. If you’re looking
+      for updates on the latest and greatest in Ember and Glimmer this is the
+      place to be. EmberFest is also a great opportunity to get in touch with
+      the European Ember Community (and friends from abroad) and hiring Ember
+      talent.
+    </p>
+    <div class="button get-involved">
+      <a alt="get involved!" href="#">Get involved!</a>
```
[the remark this answers is not in the document]
Comment (marcoow) [reply _includes/hero.html:27]: I can do it when everything else is ready

[the remark this answers is not in the document]
Comment (marcoow) [reply _includes/hero.html:27]: Please use a `<button>` element for this though

**Comment by oscard0m** [comment]
@davidhajba, we are already applying a background colour for the document's body with grey. I see it correctly in Chrome. In Safari, it shows the wrong background. I don't know why. When refreshing Safari, the background color is displayed correctly. I suggest creating an individual bug for this.

File: _includes/hero.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (oscard0m) [reply _includes/hero.html]: Considering this should be a link to the bottom of the page (#anchor-link), should this be a `<a>`, right?

File: _includes/about.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  <div class="container video-2022">
+    <h4>Ember &#123;&#123;fest&#125;&#125; 2022</h4>
+    <iframe
+      src="https://www.youtube.com/embed/irL80bQ8dHE"
+      frameborder="0"
+      allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+      allowfullscreen
+    ></iframe>
+    <p>
+      While we're getting ready for EmberFest 2023 in Madrid, check out the
```
Comment (marcoow) [inline _includes/about.html:29]:
```suggestion
      While we're getting ready for EmberFest 2023, check out the
```

We're not announcing the host city yet

File: _includes/past-speakers.html [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+					</a>
+					<a class="icon-link external-link" href="{{ speaker.web }}">
+						<img alt="link icon" src="/images/icons/link.svg" />
+					</a>
+				</div>
+			</div>
+		</li>
+		{% endfor %}
+	</ul>
+	<button href="" alt="Submit a Talk">Submit a Talk</button>
```
Comment (marcoow) [inline _includes/past-speakers.html:21]: this should link to https://www.papercall.io/emberfest-2023

File: 2022/index.html [hunk 5]
```diff
@@ -0,0 +1,246 @@
...
+            <img src="/2022/images/organizers/mainmatter.svg" />
+          </div>
+          <h4>Mainmatter</h4>
+          <div class="links">
+            <a href="https://twitter.com/_mainmatter"><img src="/2022/images/icons/twitter-blue.svg" /></a>
+            <a href="https://mainmatter.com/">
+              <img src="/2022/images/icons/link-blue.svg" />
+            </a>
+          </div>
+        </li>
```
Comment (marcoow) [inline 2022/index.html:134]: let's put us in the middle

File: 2022/index.html [hunk 6]
```diff
@@ -0,0 +1,246 @@
...
+            <input
+              type="text"
+              name="MMERGE5"
+              value="" />
+          </label>
+        </div>
+        <button type="submit" class="button">
+          Subscribe <img src="/2022/images/icons/arrow-right.svg" />
+        </button>
+      </form>
```
Comment (marcoow) [inline 2022/index.html:243]: This form doesn't seem to work properly? Try filling in values and submitting – the fields in the Mailchimp form are empty.

File: _includes/sidepages_header.html [hunk 7]
```diff
@@ -0,0 +1,18 @@
+<div class="sidepages__header">
+  <picture>
+    <source media="(max-width: 1100px)" srcset="/images/logo_horizontal.svg" />
+    <source media="(min-width: 1101px)" srcset="/images/logo_vertical.svg" />
+    <img class="logo" src="/images/logo_horizontal.svg" alt="emberfest logo" />
+  </picture>
```
Comment (marcoow) [inline _includes/sidepages_header.html:6]: the logo should link back to the homepage

**Review by marcoow — CHANGES_REQUESTED** [review]
Looks great overall! There's a few things left to change still, e.g. Papercall, ti.to, etc.

Also, the layout doesn't seem to work correctly for all mobile viewports?

app_code-of-conduct

netlify app_(Galaxy Fold)

File: 2022/index.html (hunk 5 above)
Comment (oscard0m) [reply 2022/index.html:134]: This is 2022. Do we change it for **2022 only**, **2023 only** or **both**?

Comment (marcoow) [reply 2022/index.html:134]: ah, sorry – meant for 2023

**Comment by oscard0m** [comment]
@davidhajba, when you have time, could you look at the `:hover` state of the buttons/links? Especially on the front page. I think we should arrive at a consensus on that.

**Comment by inesilva** [comment]
@davidhajba 
image
which text do you mean here?

**Comment by oscard0m** [comment]
> @davidhajba image which text do you mean here?

I think he means the placeholder text.

File: imprint/index.html [hunk 8]
```diff
@@ -1,27 +1,42 @@
...
-      <h4>Exclusion of Liability </h4>
+      <h3 >Imprint</h3>
+      <div class="imprint__responsible_info">
+        <p>
+            Publisher information according to §6 Teledienstgesetz (TDG)<br />
+            Responsible according to §6 MDStV:
+          </p>
+          <p>
+            EmberFest UG <br />
+            +49 89 452 139 03‬ <br />
```
Comment (marcoow) [inline imprint/index.html:20]:
```suggestion
            +49 89 452 139 03<br />
```

**Comment by davidhajba** [comment]
Thank you for your amazing work @oscard0m @inesilva 
Things start to look fantastic !!!! 🎉

few things 

 FRONTPAGE
 
* this should not be wider then 650px
239499011-53882474-5fb0-41cd-8d9c-4b5136e7cab8

* as this content became shorted than in Sketch center alignment would be better (here and on mobile too as @oscar suggested)
Screenshot 2023-05-22 at 09 46 03

* please center align the content vertically here 
Screenshot 2023-05-22 at 09 55 44

* the distance here should not be bigger than 200px 
Screenshot 2023-05-22 at 10 01 16

* [HERE](https://www.sketch.com/s/150ed20a-d7c3-46a1-8151-89fba683acae/v/yxvQGy/p/BE993FE2-E578-440C-A931-8457C6430A8A/canvas?posX=-968.6367104980156&posY=-217.20209659788986&zoom=0.6473674774169922) are the hover actions of the social media buttons

Currently on the website the twitter yellow button the outline of the bird is a but distorted, and both yellow and blue button it doesn't look centered. I know it's positioned center, but because the shape of the bird it's tricky, so I saved all social media buttons [HERE](https://shared-assets.adobe.com/link/d8e276af-ef77-4cce-5755-9196ee124d9a)  as SVGs I think this is the easiest way to solve this. pls add the shadow to the blue buttons

CODE OF CONDUCT, CANCELLATION POLICY, IMPRINT

* the size of the SVG background element is not the same as on the Front page, could you please make it same size and position? 

Screenshot 2023-05-22 at 10 26 35
Screenshot 2023-05-22 at 10 26 22

**Review by marcoow — APPROVED** [review]
🎉

**Comment by marcoow** [comment]
Only need to replace the video with the actual one from 2022 and add the sponsor prospectus once that's done…

**Comment by oscard0m** [comment]
> Only need to replace the video with the actual one from 2022 and add the sponsor prospectus once that's done…

Awesome! Just to be sure: Is that on @davidhajba's backlog? @marcoow

**Comment by oscard0m** [comment]
> don't think there's anything to do for David there?

@marcoow, where can we get the video and the prospectus?

**Comment by marcoow** [comment]
I'll upload here once ready

**Comment by marcoow** [comment]
here's the sponsor prospectus: 
[emberfest-2023-sponsorship-prospectus.pdf](https://github.com/emberfest/emberfest.github.io/files/11551179/emberfest-2023-sponsorship-prospectus.pdf)

**Comment by marcoow** [comment]
…and the video: https://youtu.be/hvg35gS1Xjo

**Review by marcoow — APPROVED** [review]

**Pull request merged**